tasks/push-disk-images/push-disk-images.yaml

```diff
@@ -83,8 +83,8 @@ spec:
                          -s true \
                          -t 14400 \
```

Comment:
The jira says "make it customizable", but the AC says "set the timeout to 1 day" - I don't remember, but I guess we probably concluded "just set a really high value" when grooming it. That way all that matters is the main release pipeline timeouts that are set in the RPA. (So could you maybe add something about this to the PR/commit description?)

But for that you also need to increase this timeout here, otherwise the internal pipeline might continue to run, but the internal-request script will timeout after 4 hours.

Reply:
Fixed this line and updated the commit description. Thanks. I asked something similar on the ticket so I will update the ticket about this